### DISCUSSION TITLE
Xray(M+H+K+A)+Caddy(N+T) 如何配置多个域名对应多个网站服务

### DISCUSSION THREAD

**Discussion by alex4814** [lxhao61/integrated-examples#176] [Q&A] [answered]
`Xray(M+H+K+A)+Caddy(N+T)` 这个方案配置完后，h2y.example.com, h3a.example.com, cdn.example.com 都会指向静态的文件服务，但我现在有两个服务想通过反代，分别用 h2y.example.com 和 cdn.example.com 域名通过 443 访问，请问在 json 里该如何设置呢？

**Comment by lxhao61** [comment, marked as answer]
见 ‘Caddy(Other Configuration) （Caddy 的特殊应用配置方法。）’ 中对应介绍及对应配置示例。
